src/bundles.sh

```diff
@@ -0,0 +1,18 @@
+export CONDUCTR_IP=$(hostname -i)
+
+conduct load conductr-haproxy
+conduct run conductr-haproxy --scale $NODE_COUNT
+
+sudo service haproxy restart
+
+sleep 5
+
+ES_COUNT=$(( NODE_COUNT < 3 ? NODE_COUNT : 3 ))
+
+conduct load conductr-elasticsearch
+conduct run conductr-elasticsearch --scale $ES_COUNT
```

Comment:
The formation template currently only really supports 3 nodes https://github.com/typesafehub/conductr-cloudformation/blob/master/conductr-cloudformation.yaml#L315
3 nodes would be consistent if we consider this a gratis-ware user tool